Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -689,3 +705,10 @@ export default class CovidCardView extends Vue {
         </v-row>
     </v-container>
 </template>
+
+<style lang="scss">
+.alert-background-color {
+    background-color: white !important;
+    align-content: center !important;
+}
+</style>
```

Comment:
Nit: Keeping the default alert styling is preferable, unless @mk08sh wants to deviate from that.

Reply:
@MikeLyttle I'm okay either way. We were using a design system that had Vuetify components. default styling is fine.